### PR TITLE
Changes need to run RDS dev site using new EIP

### PR DESCRIPTION
**Pull request by cgolubi1**
* Partially addresses #2855 
* This PR is what's needed to create new instances via vagrant that use RDS databases --- it's a noop for instances that use local mysql

### CODE REVIEW

File: deploy/vagrant/manifests/init.pp [hunk 1]
```diff
@@ -11,18 +11,10 @@
...
       $database_fqdn = "buttonmen-cgolubi1-2523-rds.cyk4kpmwmefe.us-east-1.rds.amazonaws.com"
     }
-    "174.129.14.204": {
-      $puppet_hostname = "alpha.buttonweavers.com"
-      $database_fqdn = "127.0.0.1"
-    }
-    "54.83.36.215": {
-      $puppet_hostname = "dev.buttonweavers.com"
-      $database_fqdn = "127.0.0.1"
-    }
```
Comment (cgolubi1) [inline deploy/vagrant/manifests/init.pp:25]: I just cleaned these up while i was here because they're no longer in use.

**Comment by cgolubi1** [comment]
Here's the list of all the steps i needed to make rds.dev.buttonweavers.com available using this code:

* Launch the BM site
* As soon as possible, tag the instance (rds.dev.buttonweavers.com in this case)
* As soon as possible, associate the rds.dev EIP with the instance
* Ctrl-C the vagrant provision when it's in "rsync"
* Run vagrant reload
* On-disk fixups from setupbmdev (not needed for staging or prod):
```
sshtmp ${ipaddr} "sudo /bin/sed --follow-symlinks -i -e '/^Config.siteType =/s/production/development/' /var/www/ui/js/Config.js"
sshtmp ${ipaddr} "sudo sed -i -e \"1 iBMSITE=$branchdir\" /var/spool/cron/crontabs/root"
```
* Run vagrant again for certbot/ssl:
```
vagrant rsync
vagrant provision
```
* Populate `/usr/local/etc/buttonmen_db_creds`
* Update the RDS DB security group to allow inbound access from the buttonmen-site security group
* At this point, mysql_root_cli should work
* Add a GRANT on the RDS DB for bmuser1 at the in-VPC IP of the instance 
* At this point, website access should work

This is probably a good blueprint for the steps i'll need for each of staging and prod.

**Comment by cgolubi1** [comment]
Successful circleci run - https://app.circleci.com/pipelines/github/cgolubi1/buttonmen/160/workflows/639f4acf-b3e2-463c-994e-d00dfd27fcb6

**Comment by blackshadowshade** [comment]
Okay, accepting this now. If we have any problems with it, I'm sure they'll turn up during deployment.

**Pull request merged**